src/transpiler.js

```diff
@@ -8,7 +8,9 @@ const transpiler = {
       phone: getPhone(source),
       email: getEmail(source),
       birthday: getBirthday(source),
-      address: getAddress(source)
+      address: getAddress(source),
+      metadata: { google: { metadata: source.metadata } },
+      resourceName: source.resourceName
```

Comment:
In the linxo konnector, we also have an id that is provided by the tier (here google, in our case linxo). I think we should try to reconciliate those 2 cases and maybe use something like `vendorId` as the name for the property.

Reply:
It would be good if metadata had a version property. It is then easier for migrations to target only the documents that needs to be updated.

Reply:
I agree the `vendorId` thing.

I didn't get the metadata version thing.

Reply:
I meant something like : 

```
{
  metadata: {
    version: 1
  }
}
```

This helps when you have a change to make to all your documents. Maybe the name of a property changes and you want to run a migration against all documents. With the version property you can target only the documents that are at an old version.